Doc/tools/templates/download.html

```diff
@@ -69,6 +64,13 @@ <h1>{% trans %}Download Python {{ dl_version }} documentation{% endtrans %}</h1>
 
 <p>{% trans %}These archives contain all the content in the documentation.{% endtrans %}</p>
 
+<p>{% trans %}
+We no longer provide pre-built PDFs of the documentation.
+To build a PDF archive, follow the instructions in the
```

Comment:
This is fine for main, but for older versions where we do have nearly up-to-date PDF versions in slightly older documentation, we should mention those, preferably with a link.